search/median_search2.cpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file median_search2.cpp
```

Comment:
No need to add the filename. It's best to let Doxygen automatically detect it, otherwise, you'd have to specify the whole directory as well.
```suggestion
 * @file
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix